packages/core/src/constants.ts

```diff
@@ -83,6 +84,8 @@ export enum RunErrorCodes {
   DocumentConfigError = 'document_config_error',
   MissingProvider = 'missing_provider_error',
   ChainCompileError = 'chain_compile_error',
+  UnsupportedProviderResponseTypeError = 'unsupported_provider_response_type_error',
+  AIProviderConfigError = 'ai_provider_config_error',
```

Comment:
Not adding these 2 errors to DB enum in this PR. I'll do in another one when storing errors